src/appkit.cs

```diff
@@ -7674,9 +7898,18 @@ partial interface NSMenu : NSCoding, NSCopying, NSAccessibility, NSAccessibility
 		[PostSnippet ("__mt_items_var = ItemArray();")]
 		void RemoveAllItems ();
 
+#if XAMCORE_4_0
+		[Export ("itemArray", ArgumentSemantic.Copy)]
+		NSMenuItem[] ItemArray { get; [Mac (10, 14, onlyOn64: true))] set; }
+#else
 		[Export ("itemArray")]
 		NSMenuItem [] ItemArray ();
 
+		[Mac (10, 14, onlyOn64: true))]
```

Comment:
one too many ')'

Reply:
The problem with bouncing between too many things at once is that I try to move too quickly.